### ISSUE TITLE
[COGNITION] Community Care second-level lists should be nested properly for assistive devices

### ISSUE THREAD

**Issue by kelsonic** [department-of-veterans-affairs/va.gov-team#2015]
## Issue
This is a follow-up ticket to [this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/484#issuecomment-534340349). The community care provider views sometimes have second-level lists that use CSS to indent them instead of nesting an `<ul>` inside the `<li>`. These lists were flagged as SC 1.3.1 issues. Screenshots attached below.

## Audit Finding
* Note 1, Defect 1

## Related Issue
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/484

## Acceptance Criteria
* As an assistive device user, I want to hear the lists announced as proper nested lists, instead of one large list.
* As a user with a level of visual acuity, I do not want to see any visual changes in the application.

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/find-locations/provider/ccp_1851813299

## Possible Fixes (optional)
This will need to include some HTML restructuring. Basic pseudo-code with no CSS consideration included here:

```html
<ul>
  <li>Primary Care: <strong>19 days</strong></li>
  <li>Specialty care:
    <ul>
      <li class="sublist">Cardiology: <strong>7 days</strong></li>
      <li class="sublist">Dermatology: <strong>5 days</strong></li>
      <li class="sublist">Mental Health: <strong>23 days</strong></li>
    </ul>
  </li>
</ul>
```

## WCAG or Vendor Guidance (optional)
* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs
![image.png](https://images.zenhubusercontent.com/5d812458c75aaf0001c430f8/4c42c850-ceb7-47fb-b4be-7f41dffed6d9)

**Comment by Karak888** [comment]
closing this ticket as it is in Prod.
